config/tipoff.php

```diff
@@ -11,7 +11,7 @@
 
     'web' => [
         'enabled' => true,
-        'uri_prefix' => '',
+        'uri_prefix' => '/',
```

Comment:
with this change other resources are going to break and fix only the weird behaviour in the `PreviewPage`, because its going to add twice the `//`

I pull this branch and test the topic resource and get 404
http://demo.test//topicos/serie
![image](https://user-images.githubusercontent.com/13080491/116154420-a5bfc800-a6b6-11eb-8241-40806b4a16c0.png)

you can check the handle method within the Actions folder for each nova resource

I solved yesterday doing this https://github.com/drewroberts/blog/pull/92/files#diff-449dc62fb466d835d4eb2fabff15684d7b6ed756fc4c36815f0c88a34bffac60

maybe is not an elegant nor cleaner solution, but does not break other resources